tsdb/db.go

```diff
@@ -973,8 +1018,22 @@ func (db *DB) run() {
 			case db.compactc <- struct{}{}:
 			default:
 			}
+		case <-oooScheduledCompact.C:
+			oooScheduledCompact.Reset(db.opts.OutOfOrderCompactInterval)
+
+			db.metrics.oooCompactionsTriggered.Inc()
```

Comment:
When increasing the metric,  you'll want:

```go
			db.metrics.compactionsTriggered.WithLabelValues("ooohead").Inc()
```
or, when increasing the regular head metrics:
```go
			db.metrics.compactionsTriggered.WithLabelValues("head").Inc()
```